dev/com.ibm.ws.jdbc.metatype/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -371,6 +371,9 @@ cursorSensitivity=Cursor sensitivity
 cursorSensitivity.desc=JDBC driver property: cursorSensitivity. Values are: 0 (TYPE_SCROLL_SENSITIVE_STATIC), 1 (TYPE_SCROLL_SENSITIVE_DYNAMIC), 2 (TYPE_SCROLL_ASENSITIVE).
 databaseName=Database name
 databaseName.desc=JDBC driver property: databaseName.
+databaseNameIgnored.desc=JDBC driver property: databaseName. This property is ignored by the driver if a URL is configured.
+databaseNameOverrides.desc=JDBC driver property: databaseName. This property will override the databaseName specified in the URL if one was configured.
+databaseNamePostgreSQL.desc=JDBC driver property: databaseName. If a URL and this property are configured, then the PostgreSQL driver will ignore this property when using type java.sql.Driver, and ignore the databaseName from the URL when using type javax.sql.DataSource.
```

Comment:
Change to:
When the PostgreSQL driver uses type java.sql.Driver and a URL and this property are configured, the PostgreSQL driver ignores this property. When the PostgreSQL driver uses type javax.sql.DataSource, and a URL and this property are configured, the PostgreSQL driver ignores the databaseName property.